src/validation.ts

```diff
@@ -0,0 +1,43 @@
+import {
```

Comment:
What is this file for exactly?

It might be useful because I know people have been quite confused by stuff like this before: https://github.com/apollographql/graphql-subscriptions/issues/169

But it should probably be a separate PR.

Reply:
I have no idea, haven't even checked if it's in @AlexLeung's original PR.  Seems to have been added by accident?